lib/v2.recommended.yaml

```diff
@@ -84,17 +83,14 @@ analyzer:
     avoid_returning_null_for_void: warning
     avoid_setters_without_getters: warning
     avoid_slow_async_io: warning
+    avoid_type_to_string: warning
     avoid_types_on_closure_parameters: warning
     avoid_unused_constructor_parameters: warning
     cascade_invocations: warning
     comment_references: warning
     control_flow_in_finally: warning
     curly_braces_in_flow_control_structures: warning
     directives_ordering: warning
-    duplicate_ignore: warning
-    invalid_required_named_param: warning
-    invalid_required_positional_param: warning
```

Comment:
Why did you remove these here? I presumed you would just remove them from the lints section.

Reply:
I moved them into `v2.yaml`

Reply:
Ah, these got lifted to required.

Reply:
What criteria do you use for what is in the base version and what is in recommended? I agree on keeping them, just curious why the move?

Reply:
I've always tried to use this: https://github.com/Workiva/workiva_analysis_options/#how-lints-and-analyzer-rules-are-evaluated

But it is still admittedly subjective. Given these are all very easy to fix and (IMO) shouldn't be very controversial, I thought required was reasonable, but I'm happy to discuss keeping any of them in recommended

Reply:
That makes sense to me, I am happy to have them required.